### PR TITLE
Set more strict requirements for Accuracy Checker

### PR DESCRIPTION
**Pull request by ilya-naumov**
Experiment to set more strict (~=) pip requirements for AC

Internal ticket 48358

### CODE REVIEW

File: tools/accuracy_checker/requirements.in [hunk 1]
```diff
@@ -1,36 +1,37 @@
 # progress bar
-tqdm
+tqdm~=4.31.1
```
Comment (eaidova) [inline tools/accuracy_checker/requirements.in:2]: why this is strict?

Comment (generalova-kate) [reply tools/accuracy_checker/requirements.in:2]: as POT dependency, see presentation option 1

Comment (eaidova) [reply tools/accuracy_checker/requirements.in:2]: I am not think that POT requirements should be preliminary on AC own (not all AC users use POT) and moreover it is old version of package (so at least it should be updated on 4.60 in POT)
cc @AlexKoff88 @MaximProshin

Comment (generalova-kate) [reply tools/accuracy_checker/requirements.in:2]: We will try to align it with NNCF  as well, it will be like this tqdm>=4.54.1 Are you ok?

Comment (eaidova) [reply tools/accuracy_checker/requirements.in:2]: yes, it is ok for me

**Review by eaidova — APPROVED** [review]

**Pull request merged**